docs/supported-versions.asciidoc

```diff
@@ -6,6 +6,7 @@
 * Beats: 7.0+
 * Elastic Agent: 7.10+ (standalone), 7.14+ (Fleet)
 * Elastic Maps Server: 7.11+
+* Helm: 3.2.0+
```

Comment:
I would put it in the fourth place to keep separate infra/tools and Elastic Stack applications.